api/src/test/java/org/pmiops/workbench/audit/AuditDataGeneratorTest.java

```diff
@@ -0,0 +1,17 @@
+package org.pmiops.workbench.audit;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+import org.pmiops.workbench.audit.synthetic.AuditDataGenerator;
+
+public class AuditDataGeneratorTest {
+
+  @Test
+  public void itGetsLongInRange() {
+    final long low = 1000L;
+    final long high = 2000L;
+    final long random = AuditDataGenerator.randomLongInRange(low, high);
+    assertThat(low < high).isTrue();
```

Comment:
what is this asserting? `1000L` will always be less than `2000L` right?

Reply:
Ugh, good catch. I'm pulling this whole data generator and test out in the next PR though. At least until I have a need to write a command to invoke it.